lib/active_admin/resource/action_items.rb

```diff
@@ -70,7 +70,7 @@ def add_default_action_items
           if controller.action_methods.include?("destroy") && authorized?(ActiveAdmin::Auth::DESTROY, resource)
             link_to(I18n.t('active_admin.delete_model', :model => active_admin_config.resource_label),
               resource_path(resource),
-              :method => :delete, :data => {:confirm => I18n.t('active_admin.delete_confirmation')})
+              :method => :delete, 'data-confirm' => I18n.t('active_admin.delete_confirmation'))
```

Comment:
Line is too long. [95/80]<br>Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.